### PR TITLE
adds english

### PR DESCRIPTION
**Pull request by Marmio64**
adds english so that humans can be even more racist to other species by excluding them from their conversations

improves rp

people who dont understand english will hear stupid words that british people like to use

also the icon is a british cup (ones found in medbay usually)

# Changelog

:cl:  
rscadd: Adds english language for humans
/:cl:

### CODE REVIEW

**Comment by Bluedazzled** [comment]
wait but i thought japenese existed-
this is really funny because felinid language

**Comment by ToasterBiome** [comment]
absolutely malding

**Comment by Mqiib** [comment]
Ok snowflake

**Comment by cuackles** [comment]
can we leave japanese to ninja only now because every race has its own lenguage ???

**Comment by maxion12345** [comment]
time for me to make a custom language just for my character :))))))))))))))))) (he's albanian)

File: code/modules/language/english.dm [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	desc = "The language of lobsterbacks. Also known as galactic common without the chinese."
+	speech_verb = "says"
+	ask_verb = "asks"
+	exclaim_verb = "exclaims"
+	key = "g"
+	flags = TONGUELESS_SPEECH
+	space_chance = 100
+	default_priority = 90
+	syllables = list(
+	"I say","good sir","schilling","mate","tosspot","bobby","lift","tele","queen",
```
Comment (tf-4) [inline code/modules/language/english.dm]:
```suggestion
	"I say","good sir","shilling","mate","tosspot","bobby","lift","telly","queen",
```

**Comment by Tipy1802** [comment]
should be french instead of english

**Comment by Tipy1802** [comment]
would be cool if humans could pick in the character menu which human language they speak (english, french, german, russian etc.)

File: code/modules/language/english.dm [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	"king","biscuit","tea","crumpet","arse","mum","trolly","quid","twat",
```
Comment (patpol4) [inline code/modules/language/english.dm:13]: blimey,quid,bloke,bloody

Comment (Marmio64) [reply code/modules/language/english.dm:13]: quid and bloke were already in there but ill add the other 2

**Review by adamsong — APPROVED** [review]

**Comment by JamieD1** [comment]
This is so fucking stupid.

**Pull request merged**